opal/mca/common/ucx/common_ucx.h

```diff
@@ -49,6 +49,11 @@ BEGIN_C_DECLS
                         __FILE__ ":" MCA_COMMON_UCX_QUOTE(__LINE__) \
                         " Error: " __VA_ARGS__)
 
+#define MCA_COMMON_UCX_WARN(...)                                    \
+    opal_output_verbose(0, opal_common_ucx.output,                  \
```

Comment:
can it call MCA_COMMON_UCX_VERBOSE(0, ..) ?

Reply:
macro is fixed